code/datums/trading/unique.dm

```diff
@@ -32,7 +32,7 @@
 	possible_wanted_items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/human      = TRADER_SUBTYPES_ONLY,
 		/obj/item/weapon/reagent_containers/food/snacks/meat/human = TRADER_THIS_TYPE,
-		/mob/living/carbon/human                                   = TRADER_ALL
+		/mob/living/carbon/human                                   = TRADER_THIS_TYPE
```

Comment:
Welp, as Lohikar said in Discord, this will require some other way of being handled. As _all_ humans and monkeys are technically of type `/mob/living/carbon/human`.